src/main/java/com/rackspace/ceres/app/model/SuggestType.java

```diff
@@ -0,0 +1,5 @@
+package com.rackspace.ceres.app.model;
+
+public enum SuggestType {
+  metrics, tagk, tagv;
```

Comment:
Style thing: can we make these all upper case?

Reply:
This is done to make it compatible with opentsdb api:- http://opentsdb.net/docs/build/html/api_http/suggest.html

Reply:
METRICS("metrics"), TAGK("tagk"), TAGV("tagv")

This should work. Mostly I like being able to easily see at a glance that something is an enum.